<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Simulador de Procesos por Lotes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(135, 135, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="capturaProcesos">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="0">
             <widget class="QLabel" name="capturatedProcessesLB">
              <property name="minimumSize">
               <size>
                <width>175</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>152</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Procesos capturados</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="processNumberSP">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="processAcountLB">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>96</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>No. Procesos</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLCDNumber" name="capturatedProcessesCounter">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>24</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
              </property>
              <property name="digitCount">
               <number>5</number>
              </property>
              <property name="value" stdset="0">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="processAcountPB">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(191, 191, 191);
border-radius: 8px;</string>
              </property>
              <property name="text">
               <string>✓</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout"/>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="dataGB">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Información de procesos</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="nameLB">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Nombre</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="nameLE">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="operationLB">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>26</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Operación</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QSpinBox" name="number1SP">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
                 </property>
                 <property name="value">
                  <number>1</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="symbolsCB">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(191, 191, 191);</string>
                 </property>
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <item>
                  <property name="text">
                   <string>+</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>-</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>*</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>/</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>% (residuo)</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="number2SP">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
                 </property>
                 <property name="minimum">
                  <number>-99</number>
                 </property>
                 <property name="value">
                  <number>1</number>
                 </property>
                 <property name="displayIntegerBase">
                  <number>10</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="idLB">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>ID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="idLE">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_15">
               <item>
                <widget class="QLabel" name="timeEstimatedLB">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Tiempo estimado</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_16">
                 <item>
                  <widget class="QSpinBox" name="timeEstimatedSP">
                   <property name="font">
                    <font>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
                   </property>
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="value">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="segsLB">
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>segundos.</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="addProcessBT">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(191, 191, 191);
border-radius: 8px;</string>
             </property>
             <property name="text">
              <string>Ok</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="programName">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Seminario de Solución de Problemas de Sistemas Operativos</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mostrarProcesos">
       <property name="maximumSize">
        <size>
         <width>1270</width>
         <height>720</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="2" column="0">
         <widget class="QGroupBox" name="processRuningGB">
          <property name="maximumSize">
           <size>
            <width>255</width>
            <height>285</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Proceso en ejecución</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QTableWidget" name="processRuningTB">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color: black;
</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
             <row>
              <property name="text">
               <string>ID</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Nombre</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Oper</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>T.Est</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>T.Trans</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>T.Res</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="globalCountLB">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Tiempo transcurrido: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="globalCountLCD">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="startRunPB">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: black;
background-color: white;
border-radius: 8px;</string>
          </property>
          <property name="text">
           <string>Comenzar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="pendientLotsLB">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>No. Lotes pendientes: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="pendientLotsLCD">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="mode">
             <enum>QLCDNumber::Dec</enum>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="lotWorkingGB">
          <property name="maximumSize">
           <size>
            <width>270</width>
            <height>221</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Lote en progreso</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QTableWidget" name="lotWorkingTB">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color: black;</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Nombre</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T.Est</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="finishedGB">
          <property name="maximumSize">
           <size>
            <width>525</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Procesos Terminados</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QTableWidget" name="finishedTB">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color: black;</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Oper</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Res</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>N.Lote</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
